newIDE/app/src/EventsSheet/EventsSearcher.js

```diff
@@ -112,15 +115,37 @@ export default class EventsSearcher extends React.Component<Props, State> {
     });
   };
 
-  _doReplaceInEvents = ({
-    searchInSelection,
-    searchText,
-    replaceText,
-    matchCase,
-    searchInConditions,
-    searchInActions,
-    searchInEventStrings,
-  }: ReplaceInEventsInputs) => {
+  _deduplicateEventSearchResults = (
+    eventsSearchResults: gdVectorEventsSearchResult
+  ) => {
+    const resultEventsWithDuplicates = mapFor(
+      0,
+      eventsSearchResults.size(),
+      i => {
```

Comment:
I don't like single letter variables, because you always wonder what is `i` when reading the callback. May I suggest `eventIndex`?